docs/Datadog.Trace.Bundle/README.md

```diff
@@ -0,0 +1,70 @@
+# `Datadog.Trace.Bundle` NuGet package
+
+This package contains the Datadog .NET APM suite. As other setups, it enables Tracing, Continuous Profiler and ASM. For tracing though, it allows both automatic and custom instrumentation. It is meant to be used mainly when access to the application server is limited (eg on Azure Service Fabric). Refer to the next section for more details on when to use this package.
+
+> If you are using automatic instrumentation and would like to interact with APM only through C# attributes, see the [Datadog.Trace.Annotations](https://www.nuget.org/packages/Datadog.Trace.Annotations/) NuGet package.
+
+## When should you use Datadog.Trace.Bundle
+
+Consider using this package mainly in scenarios where other setups aren't possible. 
+For instance, [on Windows, please install the tracer using our MSI](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=windows#install-the-tracer), on [linux, using the appropriate package](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=linux#install-the-tracer), on AAS Windows, rely on the [AAS extension](https://docs.datadoghq.com/serverless/azure_app_services/?tab=net). 
+
+Also note, that `Datadog.Trace.Bundle` will not allow you to trace apps running in IIS.
```

Comment:
I have some suggestions:

- As the very first point I think we should have a single sentence description, easily parsed, of what it contains.
- I think we should then immediately lay out the scenarios when you should or shouldn't use it.
- I put don't use it first, as seems more important.

```suggestion
This package contains the full Datadog .NET APM suite for Tracing (automatic, and custom), Continuous Profiler and Application Security Monitoring (ASM).

## Should I install this package?

Do **not** install this package if any of the following are true:

- You have already installed the [Datadog .NET Tracer MSI](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=windows#install-the-tracer) on Windows. Install [Datadog.Trace](https://www.nuget.org/packages/Datadog.Trace) instead if you need custom instrumentation.
- You have already installed the [Datadog .NET package on Linux](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=linux#install-the-tracer). Install [Datadog.Trace](https://www.nuget.org/packages/Datadog.Trace) instead if you need custom instrumentation.
- You instrument your application using the .NET dd-trace tool. Install [Datadog.Trace](https://www.nuget.org/packages/Datadog.Trace) instead if you need custom instrumentation.
- Your application is running in Azure App Services. Install [the AAS extension](https://docs.datadoghq.com/serverless/azure_app_services/?tab=net) instead.
- Your application is running in IIS on Windows. This package is not supported; [install the MSI](https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/dotnet-core/?tab=windows#install-the-tracer) instead, and install [Datadog.Trace](https://www.nuget.org/packages/Datadog.Trace) if you also need custom instrumentation.

Install this package if:

- You are running in a scenario where you can't install an MSI or Linux package. Services such as Azure Service Fabric provide limited access to the underlying server.
- You want to use both custom and automatic instrumentation, and keeping tracer versions in-sync is a chore.
- You are running multiple applications on a server and some of them use custom instrumentation, as well as automatic instrumentation.
- You prefer to deploy a static set of binaries rather than relying on existing infrastructure on the host.

> If you are using automatic instrumentation and would like to interact with APM only through C# attributes, see the [Datadog.Trace.Annotations](https://www.nuget.org/packages/Datadog.Trace.Annotations/) NuGet package.
```